packages/luals-annotations/src/luaLSType.ts

```diff
@@ -0,0 +1,59 @@
+import {
+	FunctionProperty,
+	ArrayProperty,
+	TableProperty,
+	isArrayProperty,
+	isFunctionProperty,
+	isTableProperty,
+	Property,
+} from "@lest/docs";
+
+export function formatProperty(param: Property, typeSeparator = ":"): string {
```

Comment:
Exposing this as `typeSeparator` seems like too much control given to the callers (it needs to know about the internals of this function)

It looks like you only use this to do `formatProperty(param, " ")` in another file. You should just not export this function and replace that code. This function should have a [single responsibility](https://www.digitalocean.com/community/conceptual-articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design#single-responsibility-principle)